fiftyone/utils/lerobot.py

```
@@ -0,0 +1,1549 @@
+
+    major = int(match.group(1))
+    if major != 3:
+        raise UnsupportedMediaReferenceVersionError(
+            "Unsupported LeRobotDataset format major %d; this importer "
+            "supports only v3.x" % major
+        )
+
+    if not isinstance(info["features"], dict) or not info["features"]:
+        raise MalformedMediaSourceError(
+            "LeRobot info.json must declare a non-empty features object"
+        )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)